pkg/testutil/util.go

```diff
@@ -79,11 +79,18 @@ func IsYamlEqual(expectation *string, actualYamlByteArray *[]byte) bool {
 	}
 
 	// Replace Timestamps that would show up as different.
-	_, err := exec.Command("sed", "-r", "-i", ReplaceTimestampRegExp, actualYamlFileName).Output()
```

Comment:
rather than constructing a path in this function, you could use a go embed of a filesystem (FS). https://pkg.go.dev/embed 

I think that would make this a bit cleaner and test binaries would still work whereas now a test binary would need to be in a specific location to execute properly.

Reply:
Again, agreed but this is a change required across all the unit tests in this and sagemaker-controller repo and we are tracking this issue.